usr/sbin/regionsrv-enabler-azure

```diff
@@ -12,6 +12,34 @@
 # Lesser General Public License for more details.
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library.
+
+"""
+Check the different states of the license type:
+    START   LICENSE CHANGED TO     CACHE   ACTION
+    BYOS-1  SLES                   N/A     Register
+    BYOS-1  SLES                   BYOS    Register
+    BYOS-1  BYOS-1                 N/A     Do nothing
+    BYOS-1  BYOS-1                 BYOS    Do nothing
+    BYOS-1  BYOS-2                 N/A     Do nothing
+    BYOS-1  BYOS-2                 BYOS    Do nothing
+
+    BYOS-2  SLES                   N/A     Do nothing
```

Comment:
Does `SLES` mean `PAYG`? These state transitions are not product specific but billing model specific, i.e. they apply equally to SLES For SAP and SLE HPC

Reply:
Yes, `SLES` represents all possible PAYG possible states, meaning `SES_SAP` for example